src/Sylius/Behat/Context/Ui/Admin/ManagingCatalogPromotionsContext.php

```diff
@@ -194,6 +222,28 @@ public function itShouldHaveCodeAndName(string $code, string $name): void
             $this->indexPage->isSingleResourceOnPage(['name' => $name, 'code' => $code]),
             sprintf('Cannot find catalog promotions with code "%s" and name "%s" in the list', $code, $name)
         );
+
+        $actions = $this->indexPage->getActionsForResource(['name' => $name]);
+        $actions->clickLink('Edit');
+    }
+
+    /**
+     * @Then /^it should apply to ("[^"]+" variant) and ("[^"]+" variant)$/
+     */
+    public function itShouldHaveRule(ProductVariantInterface $firstVariant, ProductVariantInterface $secondVariant): void
```

Comment:
```suggestion
    public function itShouldApplyToVariants(ProductVariantInterface ...$variants): void
```
and of course adjust the content of this method